doc/api/tls.md

```diff
@@ -792,8 +792,9 @@ changes:
   * `servername`: {string} Server name for the SNI (Server Name Indication) TLS
     extension.
   * `checkServerIdentity(servername, cert)` {Function} A callback function
-    to be used when checking the server's hostname against the certificate.
-    This should throw an error if verification fails. The method should return
+    to be used (instead of the builtin `tls.checkServerIdentity` function)
```

Comment:
Tiny nit: can you use `tls.checkServerIdentity()`? It seems to be the dominant style.

Reply:
of course! :) Updated